gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
```

Comment:
Yes, the project is not buildable with the specified version using the wrapper. I decided it wouldn't hurt to bump it here so that contributors would be able to build

Reply:
The build failure might be related to incompatible `binary-compatibility-validator` version.